dashboards/google-microservices/microservices-grpc-monitoring-gce.json

```diff
@@ -49,30 +50,25 @@
         "width": 24,
         "widget": {
           "title": "Client Concurrency",
-          "id": "",
           "xyChart": {
             "chartOptions": {
               "displayHorizontal": false,
-              "mode": "COLOR",
-              "showLegend": false
+              "mode": "COLOR"
             },
             "dataSets": [
               {
-                "breakdowns": [],
-                "dimensions": [],
-                "legendTemplate": "",
-                "measures": [],
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "outputFullDuration": false,
-                  "timeSeriesQueryLanguage": "fetch gce_instance\n  | { t_0:\n      metric 'custom.googleapis.com/opencensus/grpc.io/client/started_rpcs'\n        | align delta(1m)\n        | every 1m\n      ; t_1:\n      metric 'custom.googleapis.com/opencensus/grpc.io/client/completed_rpcs'\n        | align delta(1m)\n        | every 1m\n        | group_by drop [metric.grpc_client_status],\n            [completed_rpcs_aggregate: aggregate(value.completed_rpcs)] }\n  | outer_join [0]\n  | value [concurrent_rpcs: sub(t_0.started_rpcs, t_1.completed_rpcs_aggregate)]\n  | filter (val() >= 0)\n",
-                  "unitOverride": ""
+                  "prometheusQuery": "(\n    sum by (opencensus_task, grpc_client_method, project_id, instance_id, zone) (\n        increase(custom_googleapis_com:opencensus_grpc_io_client_started_rpcs{\n            monitored_resource=\"gce_instance\"\n        }[1m])\n    )\n    -\n    sum by (opencensus_task, grpc_client_method, project_id, instance_id, zone) (\n        increase(custom_googleapis_com:opencensus_grpc_io_client_completed_rpcs{\n            monitored_resource=\"gce_instance\"\n        }[1m])\n    )\n)\nand\n(\n    sum by (opencensus_task, grpc_client_method, project_id, instance_id, zone) (\n        increase(custom_googleapis_com:opencensus_grpc_io_client_started_rpcs{\n            monitored_resource=\"gce_instance\"\n        }[1m])\n    )\n    -\n    sum by (opencensus_task, grpc_client_method, project_id, instance_id, zone) (\n        increase(custom_googleapis_com:opencensus_grpc_io_client_completed_rpcs{\n            monitored_resource=\"gce_instance\"\n        }[1m])\n    )\n) >= 0\n\n"
```

Comment:
Just checking my understanding here... if I pretty print this it comes out looking like `(A - B) and (A - B) >= 0` right?

Does this differ from just `(A - B) >= 0`?